docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -52,20 +52,25 @@ For more information on staging, see the [staging support](#staging-support) sec
 
 ## Supported file formats
 * [insert-values](../file-formats/insert-format.md) is used by default
-* [jsonl](../file-formats/jsonl.md) supported when staging is enabled. **Note**: Currently loading compressed jsonl files is not supported. `data_writer.disable_compression` should be set to `true` in dlt config
+* [jsonl](../file-formats/jsonl.md) supported when staging is enabled (see limitations below)
 * [parquet](../file-formats/parquet.md) supported when staging is enabled
 
+The `jsonl` format has some limitations when used with Databricks:
+
+1. Compression must be disabled to load jsonl files in databricks. Set `data_writer.disable_compression` to `true` in dlt config when using this format.
+2. The following data types are not supported when using `jsonl` format with `databricks`: `decimal`, `complex`, `date`, `binary`. Use `parquet` if your data contains these types.
+3. `bigint` data type with precision is not supported with `jsonl` format
+
```

Comment:
@sh-rp @rudolfix  
Json support is even more limited. All these came up in `tests/load/pipeline/test_stage_loading.py::test_all_data_types`   
* `binary` in base64 does not work, not sure if it works at all in some other encoding.  
* Databricks parses `date` strings to timestamp and fails  
* There is no general `JSON` or equivelent type (I think), you need to create a struct column with a defined schema. And json objects/arrays don't convert automatically to string  
* Somehow a 16 bit int in JSON is detected as LONG

Reply:
I definately think we should contact databricks about this and see what they say. @rudolfix who do we know there?